### PR TITLE
add support for grpc-web_out

### PR DESCRIPTION
**Pull request by ppiccolo**

### CODE REVIEW

**Comment by mhamrah** [comment]
I'll push these momentarily

File: all/entrypoint.sh [hunk 1]
```diff
@@ -181,6 +180,9 @@ plugins=grpc+embedded\
     "node")
         GEN_STRING="--grpc_out=$OUT_DIR --js_out=import_style=commonjs,binary:$OUT_DIR --plugin=protoc-gen-grpc=`which grpc_${PLUGIN_LANG}_plugin`"
         ;;
+    "web")
+        GEN_STRING="--grpc-web_out=import_style=typescript,mode=grpcwebtext:$OUT_DIR --js_out=import_style=commonjs:$OUT_DIR --plugin=protoc-gen-grpc-web=`which grpc_${PLUGIN_LANG}_plugin`"
```
Comment (nycdotnet) [inline all/entrypoint.sh]: I see what you did there...

**Comment by Fadelis** [comment]
Is it possible to update LATEST image in docker hub to include this?

**Comment by mhamrah** [comment]
apologies, i never got a chance to push this. doing it within the hour.

**Comment by mhamrah** [comment]
pushed. I also have #101 which will bump latest to 1.17

**Pull request merged**